src/server.ts

```diff
@@ -1,10 +1,34 @@
 import { ApolloServer } from '@apollo/server';
 import { startStandaloneServer } from '@apollo/server/standalone';
 
-const typeDefs = `#graphql  
-  type Query {    
+interface User {
+  name: string;
+  email: string;
+  birthDate: string;
+}
+
+const typeDefs = `
+  type Query {
     hello: String
-  }  
+  }
+
+  type User {
+    id: ID!
+    name: String!
+    email: String!
+    birthDate: String!
+  }
+
+  input UserInput {
+    name: String!
+    email: String!
+    password: String!
+    birthDate: String!
+  }
+
+  type Mutation {
+    createUser(data: UserInput!): User
+  }
```

Comment:
Agora que você fez a primeira mutation, talvez dê pra enxergar que tanto os typeDefs quando os resolvers são valores que vão crescer conforme você for desenvolvendo, certo? Olhando um pouco pros valores do SOLID, talvez seja legal separá-los 1 em cada módulos no seu código?